simulator-samples/sample-jms-fax/src/main/resources/application.properties

```diff
@@ -16,6 +16,10 @@ citrus.simulator.defaultScenario=FaxQueued
 # Should Citrus validate incoming messages on syntax and semantics
 citrus.simulator.templateValidation=true
 
+# Number of available threads for parallel scenario execution
+citrus.simulator.executor.threads=10
+
+
```

Comment:
I am not sure if we want to have this in each sample. Imho I think it's superfluous, if the simulator comes with a default of 10. It gives a feeling that one **has** to configure it, which is not true. The doc mentions it for customers who want to configure it specifically.

```suggestion
```

Reply:
True.